src/table/components/head/MenuList/MenuList.tsx

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable import/no-cycle */
+import React from 'react';
+import Menu from '@mui/material/Menu';
+import { PopoverOrigin } from '@mui/material';
+import { MenuItemGroup } from '../../../types';
+import MenuGroupWrapper from './MenuGroupWrapper';
+
+interface MenuListProps {
+  open: boolean;
+  anchorEl: HTMLDivElement | null;
+  onClose: () => void;
+  menuGroups: MenuItemGroup[];
+  transformOrigin?: PopoverOrigin;
+  anchorOrigin?: PopoverOrigin;
+}
+
+const MenuList = ({
+  anchorEl,
+  open,
+  onClose,
+  menuGroups,
+  transformOrigin = { horizontal: 'left', vertical: 'top' },
+  anchorOrigin = { horizontal: 'left', vertical: 'bottom' },
```

Comment:
what we are giving to `MenuList` as **default values** -> it makes dropdown open from **bottom left**
but in case of sub menu -> it should open from **top right**

Actually, we can remove them! it still picks up default provided mui values and will work correctly!